src/whyDidYouRender.js

```diff
@@ -71,6 +71,15 @@ function trackHookChanges(hookName, { path: hookPath }, hookResult) {
     });
 
     if (notification.reason.hookDifferences) {
+      if (wdyrStore.options.showLineHook) {
+        try {
+          throw new Error().stack.split(' at ')
+            .find(line => line.match(displayName)).split('/').pop();
```

Comment:
From my experience this will throw an error for anonymous functions like

const MemoizedComponent = React.memo(() => {/* component stuff */});
MemoizedComponent.whyDidYouRender = true;

Reply:
Yeah, using react.memo cause an error, `Cannot read property 'split' of undefined`.

Using the position works because of no need for the match name.
`const matchedLine = new Error().stack.split(' at ')[3].split('/').pop();`

Wich approach you think is better?